### PR TITLE
feat: add command 'logs' to cli, to fetch logs from pods

### PR DESCRIPTION
**Pull request by Tanguyabel**
This PR brings a new command to the cli to fetch logs from a selected pod, making it easier to troubleshoot failing runs.
This makes subprocess calls under the hood and assumes that credentials are properly set to access the cluster.

---

- [x] This PR references an open issue
- [ ] Added a test
- [ ] Updated the documentation
- [ ] Added a screenshot of a successful customer deployment

### CODE REVIEW

File: now/cli/parser.py [hunk 1]
```diff
@@ -129,6 +129,15 @@ def set_survey_parser(sp):
     )
 
 
+def set_logs_parser(sp):
+    sp.add_parser(
+        'logs',
+        help='Fetch logs from a running pod.',
+        description='Fetch logs from a running pod.',
```
Comment (azayz) [inline now/cli/parser.py:136]: maybe in description we should add more info than just repeating same info in help

**Review by azayz — COMMENTED** [review]
Great PR, LGTM after comment

**Review by jupyterjazz — APPROVED** [review]
Nice feature!

File: now/run_all_k8s.py [hunk 2]
```diff
@@ -114,3 +115,59 @@ def start_now(**kwargs):
...
+    flow_id = flow['id']
+    _result = status_wolf(flow_id)
+    if _result is None:
+        print(f'❎ Flow not found in JCloud. Likely, it has been deleted already')
+
+    if _result is not None and _result['status']['phase'] == FLOW_STATUS:
+        namespace = _result["spec"]["jcloud"]["namespace"]
+
+    cmd = f"kubectl get pods -n {namespace}"
+    proc = subprocess.run(cmd.split(" "), stdout=subprocess.PIPE, check=True)
```
Comment (joschkabraun) [inline now/run_all_k8s.py:150]: we also have a `cmd` function defined in helper class

File: now/run_all_k8s.py [hunk 3]
```diff
@@ -114,3 +115,59 @@ def start_now(**kwargs):
...
+            'name': 'pod',
+            'message': 'Which pod do you want to check logs for?',
+            'choices': pods,
+        }
+    ]
+    pod = maybe_prompt_user(questions, 'pod', **kwargs)
+
+    container = "gateway" if "gateway" in pod else "executor"
+    cmd = f"kubectl logs {pod} -n {namespace} -c {container}"
+    subprocess.run(cmd.split(" "), check=True)
```
Comment (joschkabraun) [inline now/run_all_k8s.py:173]: similarly here

**Review by joschkabraun — COMMENTED** [review]
Nice feature! Small comment

File: now/run_all_k8s.py (hunk 2 above)
Comment (Tanguyabel) [reply now/run_all_k8s.py:150]: good catch, I am switching to it!

**Review by joschkabraun — APPROVED** [review]
LGTM!

**Pull request merged**